domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidsforhold/impl/InntektsmeldingRegisterTjeneste.java

```diff
@@ -121,6 +121,17 @@ private void logInntektsmeldinger(BehandlingReferanse referanse, Map<Arbeidsgive
         });
     }
 
+    /**
+     * Liste av alle påkrevde inntektsmeldinger
+     * inntektsmelding. Filtrert ut åpenbart passive arbeidsforhold
+     */
+    public Map<Arbeidsgiver, Set<InternArbeidsforholdRef>> hentAllePåkrevdeInntektsmeldinger(BehandlingReferanse referanse) {
+        Objects.requireNonNull(referanse, VALID_REF);
+        var inntektArbeidYtelseGrunnlag = inntektArbeidYtelseTjeneste.finnGrunnlag(referanse.behandlingId());
+        var påkrevdeInntektsmeldinger = utledPåkrevdeInntektsmeldingerFraGrunnlag(referanse, inntektArbeidYtelseGrunnlag);
+        return filtrerInntektsmeldingerForYtelseUtvidet(referanse, inntektArbeidYtelseGrunnlag, påkrevdeInntektsmeldinger);
```

Comment:
Bør du ikke ha både filtrerInntektsmeldingerForYtelse og filtrerInntektsmeldingerForYtelseUtvidet ?

Reply:
Gjør ikke disse metodene det samme? Hvis vi ser SVP implemenasjon av  filtrerInntektsmeldingerForYtelse så filtrerer denne ut de det ikke er søkt tilrettelegging for. Mens filtrerInntektsmeldingerForYtelseUtvidet filtrerer på inaktive forhold i tillegg til de det er søkt tilrettelegging for. Jeg forstår ikke helt hvorfor man bruker begge andre steder?

Reply:
Så lenge siden jeg holdt på med de greiene. 
Nå ser det ut til at vi kan bruke filtrerInntektsmeldingerForYtelse() kun for sjekk før INREG og at utledManglendeInntektsmeldingerFraGrunnlag klarer seg med filtrerInntektsmeldingerForYtelseUtvidet(). 
Kan jo bytte navn på dem til filtrerInntektsmeldingerForYtelseKunAAREG og filtrerInntektsmeldingerForYtelseMedGrunnlag